src/command/tools/merge.rs

```diff
@@ -0,0 +1,114 @@
+use apollo_parser::ast;
+use apollo_parser::Parser as ApolloParser;
+use clap::Parser;
+use serde::Serialize;
+
+use std::fs;
+use std::path::{Path, PathBuf};
+use std::ffi::OsStr;
+
+use crate::options::ToolsMergeOpt;
+use crate::{RoverOutput, RoverResult};
+
+#[derive(Clone, Debug, Parser, Serialize)]
+pub struct Merge {
+    #[clap(flatten)]
+    options: ToolsMergeOpt,
+}
+
+impl Merge {
+    pub fn run(&self) -> RoverResult<RoverOutput> {
+        // find files by extension
+        let schemas = self.find_files_by_extensions(self.options.schemas.clone(), &["graphql", "gql"])?;
```

Comment:
I think some folks also use `.graphqls` extension for schemas and `.graphql` for operation

Reply:
good catch. might be worth allowing users to use a custom extension as a flag in that case